### DISCUSSION TITLE
Is it possible to implement response.iter_content() with pyfetch?

### DISCUSSION THREAD

**Discussion by marianoweber** [pyodide/pyodide#2613] [General]
Hey,

I was wondering if it is technically possible to implement something like `response.iter_content()` which `requests` has, into `pyfetch`?

Thank you!

**Comment by hoodmane** [comment]
Yeah I think it's technically possible using `getReader`:
https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream/getReader
